src/rust/cryptography-cffi/build.rs

```diff
@@ -69,6 +69,10 @@ fn main() {
         .flag_if_supported("-Wno-error=sign-conversion")
         .flag_if_supported("-Wno-unused-parameter");
 
+    if let Some(out_dir_str) = Path::new(&out_dir).to_str() {
+        build.flag_if_supported(format!("-fmacro-prefix-map={}=.", out_dir_str).as_str());
```

Comment:
Can you please add a comment explaining what this is doing.

Reply:
Added